amnestywebsite/humanity-theme
# Header image credits shown on video

Ref: https://github.com/amnestywebsite/humanity-theme/issues/2

Adds a conditional to make sure the featured image caption is not shown the type is video

**Steps to test**:
1. Add a Header block to a post
2. Populate with content and add a featured image (ensure the image has captions and credits etc)
3. Switch the background type to video and assign a video
4. Save and view the front end
5. The image caption and credit should not display on the video

Example: http://bigbite.im/v/eQd81q
